lib/flop_phoenix/table.ex

```diff
@@ -260,10 +261,12 @@ defmodule Flop.Phoenix.Table do
   attr :on_sort, JS
   attr :target, :string
 
+  slot :inner_block
+
   defp sort_link(%{on_sort: nil, path: path} = assigns)
        when is_binary(path) do
     ~H"""
-    <.link patch={@path}>{@label}</.link>
+    <.link patch={@path}>{@label}{render_slot(@inner_block)}</.link>
```

Comment:
I think we'll need to wrap the label into a `<span>` here and below, or else it will be difficult to style the link contents. For example, you might want to change the order of label/arrow in rtl languages by turning the link into a flex box. The arrow is already wrapped in a span.